scripts/requirements-dev.txt

```diff
@@ -0,0 +1,16 @@
+pre-commit
+
+# formatting
+black
+isort
+
+# testing
+flake8 == 3.6.0
```

Comment:
Is there particular reason we add spaces before and after `==` ? It doesn't matter: I just haven't seen it before.

Reply:
Thanks, applied your feedback.